modules/preferences/main/index.coffee

```diff
@@ -207,7 +207,10 @@ class Preferences extends hx.EventEmitter
       modal: modal
     }
 
-    @locale moment?.locale() or navigator.language or 'en'
+    defaultLocaleId = moment?.locale() or navigator.language or 'en'
```

Comment:
probably doesn't need the `or 'en'` on this line as navigator.language will (afaik) always be defined